netci.groovy

```diff
@@ -326,19 +326,28 @@ def targetGroupOsMapInnerloop = ['netcoreapp': ['Windows_NT', 'Ubuntu14.04', 'Ub
                     }
                 }
 
-                // Set up triggers
-                if (isPR) {
-                    // We run Tizen Debug and Linux Release as default PR builds
-                    if ((osName == "Tizen" && configurationGroup == "Debug") || (osName == "Linux" && configurationGroup == "Release")) {
-                        Utilities.addGithubPRTriggerForBranch(newJob, branch, "${osName} ${abi} ${configurationGroup} Build")
-                    }
-                    else {
+                // Disable Tizen except when explicitly requested. See corefx/issues/28901
+                if (osName == "Tizen") {
```

Comment:
I think this is wrong; see the outerloop at L283-L286 where Tizen will be passed as osName twice (once for Debug and once for Release)